rdrf/explorer/utils.py

```diff
@@ -58,6 +57,10 @@ def __init__(self, form_object=None, verify=False):
             for r in CDEPermittedValue.objects.values('code', 'value', 'pv_group_id')
         }
 
+        self.forms_mapping = {f.name: f for f in RegistryForm.objects.filter(registry=self.registry_model)}
```

Comment:
Can we cache the queryset while preserving django querying functionality like so:

```python
self.forms = RegistryForm.objects.filter(registry=self.registry_model)
...
form_model = self.forms.get(name=cde_dict["formName"], registry=self.registry_model)
```

Reply:
Seems like that's not the case. Tried that in my local shell and watched for new sql queries. It doesn't cache the result that way....

https://docs.djangoproject.com/en/2.2/topics/db/queries/#when-querysets-are-not-cached